packages/react/src/components/Tree/Tree.tsx

```diff
@@ -102,10 +102,14 @@ class Tree extends AutoControlledComponent<WithAsProp<TreeProps>, TreeState> {
   }
 
   computeNewIndex = (index: number) => {
-    const { exclusive } = this.props
+    const activeIndexes = this.getActiveIndexes()
+    if (!this.props.items[index]['items']) {
```

Comment:
can an item be undefined?
also, can you cast `items[index]` to `TreeItemProps`? In this way we're protected in case the API for `TreeItem` changes when we access `items[index].items`

Reply:
no, it shouldn't be undefined, since this is called by an item click handler.
will refactor this a bit, you can take a look.